src/preview.js

```diff
@@ -26,88 +47,52 @@ const getPreviewHeader = ( lang, imageUrl = '' ) =>{
 					<span class="wikipediapreview-footer-cta wikipediapreview-footer-cta-readmore">${msg( lang, 'continue-reading' )}</span>
 					<a href="${buildWikipediaUrl( lang, data.title, isTouch )}" class="wikipediapreview-footer-cta wikipediapreview-footer-cta-readonwiki" target="_blank">${msg( lang, 'read-more' )}</a>
 				</div>
-			</div>
-	`.trim()
+			`.trim()
+
+		return render( lang, isTouch, data.dir, getPreviewHeader( lang, imageUrl ), bodyContent )
 	},
 
 	renderLoading = ( isTouch, lang, dir ) => {
-		return `
-			<div class="wikipediapreview${isTouch ? ' mobile' : ''}" lang="${lang}" dir="${dir}">
-					${getPreviewHeader( lang )}
-					<div class="wikipediapreview-loading">
-						<div class="wikipediapreview-body">
-							<div class="wikipediapreview-loading-body-line larger"></div>
-							<div class="wikipediapreview-loading-body-line medium"></div>
-							<div class="wikipediapreview-loading-body-line larger"></div>
-							<div class="wikipediapreview-loading-body-line medium"></div>
-							<div class="wikipediapreview-loading-body-line smaller"></div>
-							<div class="wikipediapreview-loading-body-line larger"></div>
-							<div class="wikipediapreview-loading-body-line medium"></div>
-							<div class="wikipediapreview-loading-body-line larger"></div>
-							<div class="wikipediapreview-loading-body-line medium"></div>
-							<div class="wikipediapreview-loading-body-line smaller"></div>
-						</div>
-					</div>
-					<div class="wikipediapreview-loading-footer"></div>
-			</div>
-  `.trim()
+		const bodyContent = `
+      <div class="wikipediapreview-loading">
+        <div class="wikipediapreview-body">
```

Comment:
the other div container built in `getPreviewBody`

```
wikipediapreview-body wikipediapreview-body-${type}
```

Why in loading we need to resuse it here?

Reply:
Hmm you are correctly pointing out an inconsistency with the loading state HTML structure compared with the other states, if I'm understanding you well. How about https://github.com/wikimedia/wikipedia-preview/pull/81/commits/a4407940f9ad727c71dc9769234a75350967c47f? I'm updating the HTML as well as preview.less so it is more consistent with the rest of the states.